### ISSUE TITLE
[Content organization] Content organization and/or categorization isn't clear or actionable.  (02.06.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#54301]
#### VFS team name
Sitewide Facilities

#### VFS product name
VAMC

#### VFS feature name
VAMC Lovell Federal Health care

#### Point of Contact/Reviewers
Sara Sterkenburg - @sterkenburgsara - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Content organization and/or categorization isn't clear or actionable. 

### Issue Details
Sections of content in a few places are missing a heading. The "additional resources" box at the base of the About Us > History page (both VA and TRICARE) and at the base of the Pallative and hospice care page (VA only)

### Link, screenshot or steps to recreate
https://web-liqqgmzl7tsyfbwyulbkuborufuuttav.demo.cms.va.gov/lovell-federal-health-care-va/programs/palliative-and-hospice-care/ and https://web-liqqgmzl7tsyfbwyulbkuborufuuttav.demo.cms.va.gov/lovell-federal-health-care-va/about-us/history/ 

### VA.gov Experience Standard
[Category Number 02, Issue Number 06](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.1

### Platform Recommendation
Add relevant headings to these boxes of content to give the user context. The links themselves should also have some explanatory text to contextualize their purpose and value to end-users.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by jilladams** [comment]
@davidmpickett same here re: Keep vs. Ignore for speedruns. 

In my experience from Collab Cycle: if this is valid / actionable and we intend to fix, we should make / link tracking tickets in our repo. If not and we never intend to fix, the product teams own closing CC tickets. Also though: a11y issue. So maybe the right starting point is: @davidmpickett did you say Ignore bc you know we're already tracking in a cms ticket? And if not: @laflannery  can opine about a11y priority related to Lovell?

**Comment by laflannery** [comment]
@jilladams From an a11y standpoint, this is all very content-y so I'm not sure how much would actually fall to your team but my comments are:

- There are no actual violations on the pages mentioned in the description
- I do have some recommendations though that would help to make these pages clearer and more understandable:
  - In the "Additional Resources" section on both pages if would definitely be better if the links, because they are H3s, had descriptions as well. There is a field for this in the CMS so it is possible for the editor to add this. Having H3s like this without any context is not ideal.
    -If we did want to look at engineering solutions a couple could be to 1. make these links not headings or 2. make the link summary required in the CMS. 
  - On the [Palliative and Hospice care page](https://www.va.gov/lovell-federal-health-care-va/programs/palliative-and-hospice-care/), the content probably doesn't need to be in accordions. It's really not that much content and just adds additional steps and cognitive load for folks.
  - On the [History pages](https://www.va.gov/lovell-federal-health-care-va/about-us/history/), the 3 accordions at the bottom of the page seem out of order. There is a nice section about Captain Lovell himself and then the descriptions about the clinics are nested within this section. This is very confusing for all users. I would suggest moving these to under the East campus heading where the clinics are first mentioned.

**Comment by davidmpickett** [comment]
@jilladams This ticket was one that fell into the category of "Content issues that Lovell editors could handle (provided someone prompts them)" per [this slack thread](https://dsva.slack.com/archives/C0FQSS30V/p1678733405952589). This may have been resolved via conversations in Lovell Office Hours or via @kmariepat-cityfriends sharing these tickets with VHA DM.

**Comment by laflannery** [comment]
@davidmpickett and @jilladams The original issue from the description was that there was no heading for "Additional Resources" on the 2 pages mentioned. I checked on live and this heading exists now. Are we able to close this ticket now?
